### PR TITLE
pkp/pkp-lib#1039 Show announcements at the published date

### PR DESCRIPTION
**Pull request by lepidus**
Fixes issue pkp/pkp-lib#1039

### CODE REVIEW

File: classes/announcement/PKPAnnouncementDAO.inc.php [hunk 1]
```diff
@@ -338,7 +338,7 @@ function &getAnnouncementsNotExpiredByAssocId($assocType, $assocId, $rangeInfo =
 			FROM announcements
 			WHERE assoc_type = ?
 				AND assoc_id = ?
-				AND (date_expire IS NULL OR date_expire > CURRENT_DATE)
+				AND (date_expire IS NULL OR date_expire >= CURRENT_DATE)
```
Comment (beghelli) [inline classes/announcement/PKPAnnouncementDAO.inc.php]: What does date_expire have to do with the publish date? I am not sure if I followed your fix, but it seems that you changed the way the expire date is working, getting announcements that would be expired if the equal operator was not used.

File: classes/announcement/PKPAnnouncementDAO.inc.php [hunk 2]
```diff
@@ -338,7 +338,7 @@ function &getAnnouncementsNotExpiredByAssocId($assocType, $assocId, $rangeInfo =
...
 				AND (date_posted < CURRENT_DATE)
```
Comment (beghelli) [inline classes/announcement/PKPAnnouncementDAO.inc.php]: I think that the fix should go here, date_posted <= CURRENT_DATE, right? And thanks for your bug report and the pull request :)

Comment (gpieri) [inline classes/announcement/PKPAnnouncementDAO.inc.php]: Whoops -- You're absolutely right, @beghelli. I cherry-picked the wrong commit from my development branch for creating the PR. Sorry. 

I'm not sure how I should proceed now. Should I update this PR or create a new one for you guys?

Comment (asmecher) [inline classes/announcement/PKPAnnouncementDAO.inc.php]: And this is my fault too for skimming the code so quickly. @gpieri, can you submit a PR that corrects the earlier one? I'll port it around.

Comment (gpieri) [inline classes/announcement/PKPAnnouncementDAO.inc.php]: Sure, @asmecher. I will do it now.

**Pull request merged**